triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/AbstractFuncTests.scala

```diff
@@ -314,21 +314,22 @@ abstract class AbstractFuncTests
     "MaxMessageSizeTests" should {
       val triggerId =
         QualifiedName.assertFromString("MaxInboundMessageTest:maxInboundMessageSizeTrigger")
-      val tId = LedgerApi.Identifier(packageId, "MaxInboundMessageTest", "MessageSize")
       "fail" in {
         for {
           client <- ledgerClient(
-            maxInboundMessageSize = 5 * RunnerConfig.DefaultMaxInboundMessageSize
+            maxInboundMessageSize = 300
```

Comment:
Why do we use 300 ? maybe a comment will be useful.

Reply:
added a comment, really anything sufficiently low works